apps/shell/tash_main.c

```diff
@@ -151,19 +151,19 @@ char *tash_read_input_line(int fd)
 #if !defined(CONFIG_DISABLE_POLL)
 		FD_ZERO(&tfd);
 		FD_SET(fd, &tfd);
-
+		tash_pm_resume();
```

Comment:
we should add this code between #ifdef CONFIG_PM #endif

Reply:
Done